app/controllers/api/service_templates_controller.rb

```diff
@@ -4,5 +4,57 @@ class ServiceTemplatesController < BaseController
     include Subcollections::Tags
     include Subcollections::ResourceActions
     include Subcollections::ServiceRequests
+
+    def create_resource(_type, _id, data)
+      # Temporarily only supporting atomic.
+      raise 'Service Type composite not supported' if data['service_type'] == 'composite'
+      create_atomic(data).tap(&:save!)
+    rescue => err
+      raise BadRequestError, "Could not create Service Template - #{err}"
+    end
+
+    private
+
+    def create_atomic(data)
+      service_template = ServiceTemplate.new(data.except('config_info'))
+      config_info = data['config_info'].except('ae_endpoints') || {}
+      case service_template.type
+      when 'ServiceTemplateOrchestration'
```

Comment:
@jntullo I am currently developing logic to create service template for the new AnsiblePlaybook service type. I think for each type we should have a class method to create a full stack of catalog item definition, meaning ServiceTemplate, ServiceResource, and ResourceActions. We can work together to move us to the same direction.